source/sass/views/initiatives.scss

```diff
@@ -5,3 +5,33 @@
 .bottom-line {
   border-bottom: 1px solid $light-gray;
 }
+
+// Fellowships--Support
+
+#view-fellows-support #multipage-nav-mobile {
```

Comment:
is this missing a comma, or should this only be `#multipage-nav-mobile`?

Reply:
No, it's not missing a comma. This is specifically to target the mobile nav for the Support Page. I spoke with @mmmavis about targeting this nav yesterday and this was the approach we came up with that won't alter all of the mobile nav components. I also made a ticket #2558 for other fellowship pages with this issue. However, the hope is that in the near future as we're cms-ifying pages we can clean this up and eliminate this sort of issue where it would need custom css.